### PR TITLE
Note about non pi os derivitives

### PR DESCRIPTION
**Pull request by mobilegmYT**
@Botspot might wanna tweak this to sound better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Introducing **Pi-Apps**, basically a list of pre-made apps you can install with
 > I want to give the devs of Pi-Apps a big shout-out."
 
 ## To install Pi-Apps
+Please note that pi-apps is made for Raspberry Pi OS and it's derivitives (Twister OS, amogOS, Ubuntu, etc). pi-apps will **NOT** work with Arch Linux, OpenSUSE, or Fedora. pi-apps is designed for RPI beginners anyway.
```
Comment (Itai-Nelken) [inline README.md:15]: Ubuntu is not based on RpiOS...

Comment (mobilegmYT) [reply README.md:15]: I'll just change it to debian ig

**Comment by Itai-Nelken** [comment]
Maybe add something like this:

Its not made for arch...pi-apps is made for beginners, and if you are using any of them, you aren't a beginner probably.

**Comment by Itai-Nelken** [comment]
Loojs good to me, just 2 typos:
Derivatives
Aren't

**Comment by mobilegmYT** [comment]
> Derivatives

According to google, aren't and derivative are both correct

**Comment by Itai-Nelken** [comment]
I just checked and I'm not wrong:

**Comment by Botspot** [comment]
Can we think of any other ***major*** debian-based Arm OS'es that are worth mentioning?

**Comment by chunky-milk** [comment]
> Can we think of any other ***major*** debian-based Arm OS'es that are worth mentioning?

Kali Linux ARM maybe?

**Comment by Botspot** [comment]
> > Can we think of any other _**major**_ debian-based Arm OS'es that are worth mentioning?
> 
> Kali Linux ARM maybe?

I already did. Is there any other debian-based OS for RPi?

**Comment by mobilegmYT** [comment]
> > > Can we think of any other _**major**_ debian-based Arm OS'es that are worth mentioning?
> > 
> > 
> > Kali Linux ARM maybe?
> 
> I already did. Is there any other debian-based OS for RPi?

I think fenix, but I'm not sure if it's debian based.

**Comment by Botspot** [comment]
> I think fenix, but I'm not sure if it's debian based.

Meh. Merging...

**Pull request merged**